types/test/vue-test.ts

```diff
@@ -96,6 +96,11 @@ class Test extends Vue {
     this.use;
     this.mixin(Test);
     this.compile("<div>{{ message }}</div>");
+    this
+      .use(() => {
+        
+      })
+      .mixin({});
```

Comment:
This only checks return type of `use`. We also need to check return type of `mixin`.